hq/app/config/Config.scala

```diff
@@ -114,6 +116,57 @@ object Config {
     config.getOptional[String]("snykSSOUrl")
   }
 
-  def getAnghammaradSNSTopicArn(config: Configuration): Option[String] = config.getOptional[String]("anghammaradSnsArn")
-  def getIamDynamoTableName(config: Configuration): Option[String] = config.getOptional[String]("iamDynamoTableName")
+  def getAnghammaradSNSTopicArn(config: Configuration): Option[String] = config.getOptional[String]("alert.anghammaradSnsArn")
+  def getIamDynamoTableName(config: Configuration): Option[String] = config.getOptional[String]("alert.iamDynamoTableName")
+  def getIamUnrecognisedUserS3Bucket(config: Configuration): Option[String] = config.getOptional[String]("alert.iamUnrecognisedUserS3Bucket")
+  def getIamUnrecognisedUserS3Key(config: Configuration): Option[String] = config.getOptional[String]("alert.iamUnrecognisedUserS3Key")
+  def getAllowedAccountIds(config: Configuration): Option[Seq[String]] = config.getOptional[Seq[String]]("alert.allowedAccountIds")
+
+  def getConfigProperties(config: Configuration)(implicit ec: ExecutionContext): Attempt[UnrecognisedJobConfigProperties] = {
+    for {
+      accounts <- getAllowedAccountsForStage(config)
+      key <- getJanusDataFileKey(config)
+      bucket <- getIamUnrecognisedUserBucket(config)
+      securityAccount <- getSecurityAccount(config)
+    } yield UnrecognisedJobConfigProperties(accounts, key, bucket, securityAccount)
+  }
+
+  def getAllowedAccountsForStage(config: Configuration): Attempt[List[String]] = {
+    Attempt.fromOption(
+      getAllowedAccountIds(config).map(_.toList),
+      FailedAttempt(Failure("unable to get list of accounts allowed to make changes to AWS. Rectify this by adding allowed accounts to config.",
+        "unable to get list of accounts allowed to make changes to AWS. Rectify this by adding allowed accounts to config.",
+        500
+      ))
+    )
+  }
+
+  def getJanusDataFileKey(config: Configuration): Attempt[String] = {
+    Attempt.fromOption(
+      Config.getIamUnrecognisedUserS3Key(config),
+      FailedAttempt(Failure("unable to get janus data file key from config for the IAM unrecognised job",
+        "unable to get janus data file key from config for the IAM unrecognised job",
+        500)
+      )
+    )
+  }
+
+  def getIamUnrecognisedUserBucket(config: Configuration): Attempt[String] = {
+    Attempt.fromOption(
+      Config.getIamUnrecognisedUserS3Bucket(config),
+      FailedAttempt(Failure("unable to get IAM unrecognised user bucket from config",
+        "unable to get IAM unrecognised user bucket from config",
+        500)
+      )
+    )
+  }
+
+  def getSecurityAccount(config: Configuration): Attempt[AwsAccount] = {
+    Attempt.fromOption(
+      Config.getAwsAccounts(config).find(_.name == "security"),
```

Comment:
Thank you so much for spotting this @jorgeazevedo, this is a bug! It was changed to use the id, instead of the name, but this part of the code mistakenly wasn't changed. Thank you!